docs/how_it_works/univariate_drift_detection.rst

```diff
@@ -82,17 +82,23 @@ The new bins relative frequency for the reference sample is set to 0.
 The relative frequency for each bin is calculated for the reference and chunk samples. Those results are then
 used to calculate the Jensen-Shannon Distance.
 
-Unlike KS D-static that looks at maximum difference between two empirical CDFs, JS distance looks at the total difference between empirical Probability Density Functions
-(PDF). This makes it more sensitive to changes that may be ignored by KS. This effect can be observed in the plot below to get the intuition:
+..
+    Unlike KS D-static that looks at maximum difference between two empirical CDFs, JS distance looks at the total difference between empirical Probability Density Functions
+    (PDF). This makes it more sensitive to changes that may be ignored by KS. This effect can be observed in the plot below to get the intuition:
```

Comment:
Why leave the two lines above? I m not sure if they are shown in the docs.

Reply:
If I get it right you left there as comments? but without a clarifier on why they are there this will be confusing two months later